vllm/model_executor/kernels/linear/scaled_mm/cutlass.py

```
@@ -166,6 +166,23 @@ def is_supported(
+            if compute_capability is not None
+            else current_platform.get_device_capability()
+        )
+        if cap is None:
+            return False, "CUDA capability could not be determined."
+
+        cap_int = cap.to_int() if hasattr(cap, "to_int") else cap
+        if not ops.cutlass_scaled_mm_supports_fp8(int(cap_int)):
+            return (
+                False,
+                "CutlassFP8ScaledMMLinearKernel is not supported for "
+                f"CUDA capability {cap}.",
```

Ruff (0.16.3)

[warning] Unparenthesized implicit string concatenation in collection

Did you forget a comma?

(ISC004)